src/utils/processorUtils.ts

```diff
@@ -0,0 +1,229 @@
+import { ProcessorRequest, ProcessorType, Source } from "@openapi/generated";
+import { EventFilter, FilterType, ProcessorFormData } from "../types/Processor";
+import { isCommaSeparatedFilterType } from "@utils/filterUtils";
+import { JSONSchema7, JSONSchema7Definition } from "json-schema";
+
+/**
+ * Prepare configuration params before reading and writing them.
+ * Its only purpose is to manage secret fields and convert to and from
+ * their masked value (empty object: {})
+ * @param data The configuration params object
+ * @param schema The json schema object
+ * @param mode "read" when retrieving data, "write" when preparing data for a save request
+ * @param [parentKey] Key of the parent configuration parameter used when drilling nested data
+ */
+export const prepareConfigParameters = (
```

Comment:
This is responsible for handling secret parameters values.
Used in particular when passing initial data to the generated form and when preparing data to be sent to processor requests.